know/rdf/parser/wikidata_formatter.py

```diff
@@ -63,3 +63,53 @@ def format_date(val: str, key: str) -> str:
         return val
 
     return format_query(response, entries_translations, entries_links, format_date)
+
+def format_book(response: dict) -> dict:
+    """ Formats a book into the expected output format
+        - Harry Potter: https://www.wikidata.org/wiki/Q8337
+        - To a Mouse: https://www.wikidata.org/wiki/Q3307327

```

Comment:
```suggestion
        - Dune: https://www.wikidata.org/wiki/Q190192
```

`To a mouse` is a bad example (since it's barely filled out and will make us look bad lol), replace it with Dune like so